watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/metab/metab_global_costing.yaml

```diff
@@ -0,0 +1,55 @@
+base_currency: USD_2020
+base_period: year
+defined_flows:
+  electricity:
+    value: 0.08
+    units: USD_2020/kWh
+  heat:
+    value: 0.0000061
+    units: USD_2020/kJ
+  H2_product:
+    value: -0.733
+    units: USD_2020/kg
+  CH4_product:
+    value: -0.305
```

Comment:
I'll double check this as well even though we could leave as is and adjust in flowsheet.